YoCodeAutomatedTests/HelpMessageTest.cs

```diff
@@ -0,0 +1,60 @@
+using Xunit;
+using FluentAssertions;
+using YoCode;
+using System.IO;
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace YoCodeAutomatedTests
+{
+    public class HelpMessageTest
+    {
+        private string TestPath;
+        private string dllPath;
+        public static IConfiguration Configuration;
+ 
+        public HelpMessageTest()
+        {
+            var builder1 = new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).AddJsonFile("testappsettings.json");
+            Configuration = builder1.Build();
+        }
+
+        [Fact]
+        public void CheckHelpMessage()
+        {
+            TestPath = Configuration["AutomatedTesting:TestDataPath"];
+            dllPath = Configuration["YoCodeLocation:DLLFolderPath"];
+
+            const string argument = "YoCode.dll --help";
+            ProcessRunner pr = new ProcessRunner("dotnet", dllPath, argument);
+            pr.ExecuteTheCheck();
+
+            var actualPath = TestPath+@"\Outputs\helpMessageActualOutput.txt";
+            var expectedPath = TestPath+@"\Outputs\helpMessage.txt";
+
+            if (!File.Exists(actualPath))
+            {
+                using (FileStream fs = File.Create(actualPath)) { }
+            }
+
```

Comment:
Lines 35-38 and line 44 below could all be replaced with `File.WriteAllText(actualPath, pr.Output)`, which would be a lot neater.